### DISCUSSION TITLE
Help: Language is not shown

### DISCUSSION THREAD

**Discussion by jejaquez** [github-linguist/linguist#6015] [Classification] [answered]
## Problem description

Linguist does not show the language of my repository.

I also ran linguist locally and it does not show the language.

```
user@66a0745ad51c:~/platform$ github-linguist .
user@66a0745ad51c:~/platform$ 
```
However, if I run linguist against a single file it works
```
user@66a0745ad51c:~/platform$ github-linguist src/frontend/app/Http/Livewire/Kb/Page.php 
src/frontend/app/Http/Livewire/Kb/Page.php: 116 lines (104 sloc)
  type:      Text
  mime type: application/x-httpd-php
  language:  PHP
user@66a0745ad51c:~/platform$ 

user@66a0745ad51c:~/platform$ github-linguist src/backend/api/src/utils.c 
src/backend/api/src/utils.c: 516 lines (450 sloc)
  type:      Text
  mime type: text/plain
  language:  C
user@66a0745ad51c:~/platform$ 
```

### URL of the affected repository:
https://github.com/mageni/mageni

### Last modified on:
2022-08-13

### Expected language:
C and PHP

### Detected language:
None

Thanks in advance for your help.

**Comment by lildude** [comment, marked as answer]
This is happening because your repo contains so many objects that you're hitting [this hard limit](https://github.com/github/linguist/blob/c497388e7f88736d7a2fe2735f64366b41a4b5f8/lib/linguist/repository.rb#L132). This is the same as https://github.com/github/linguist/discussions/5878 and other discussions referenced in it.

**Reply by jejaquez** [reply]
Thank you @lildude
